packages/plugins/ui-theme/types/packageMeta.ts

```diff
@@ -25,6 +27,9 @@ export interface PackageMetaInterface {
       type?: string;
       url?: string;
     };
+    main?: string;
+    type?: ModuleType;
+    types?: string;
```

Comment:
This is a nice feature for sure.

Question about how officially recognized the `types` field is?

I see it listed in https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html, but not in https://docs.npmjs.com/cli/v8/configuring-npm/package-json.

Reply:
If there is `"types": "./index.d.ts",` on the package.json, yeah it is not an standard and seems open field, but well we can work with that, one day you will find slide of pizza within it and will be fine 🤷🏼‍♂️